<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>[pi] server cmd started saying this</title>
    <link>https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851</link>
    <description>so when i log out the client screen freezes and the server cmd says this. the player char file will save and all but it freezes and you have to restart client for you to log back on.


[code]acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsEx
ception: Communications link failure

The last packet successfully received from the server was 1,287,290,926,062 mill
iseconds ago.  The last packet sent successfully to the server was 0 millisecond
s ago.
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance
(Unknown Source)
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(Unknown Source)
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException
(SQLError.java:1074)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:341)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI
mpl.java:2137)
[10/17/10 12:48 AM]:    ... 20 more
[10/17/10 12:48 AM]: Caused by: java.net.ConnectException: Connection refused: c
onnect
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.socketConnect(Native Method)

[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.doConnect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connectToAddress(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.SocksSocketImpl.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.&lt;init&gt;(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.&lt;init&gt;(Unknown Source)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.StandardSocketFactory.connect(Standard
SocketFactory.java:253)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:290)
[10/17/10 12:48 AM]:    ... 21 more
[10/17/10 12:48 AM]: java.lang.NullPointerException
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:29)
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:53)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: java.lang.NullPointerException
[10/17/10 12:48 AM]:    at server.util.SQL.destroyConnection(SQL.java:42)
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:33)
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:54)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Co
mmunications link failure

The last packet successfully received from the server was 1,287,290,927,046 mill
iseconds ago.  The last packet sent successfully to the server was 0 millisecond
s ago.
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance
(Unknown Source)
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(Unknown Source)
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException
(SQLError.java:1074)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI
mpl.java:2214)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.&lt;init&gt;(ConnectionImpl.j
ava:773)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.JDBC4Connection.&lt;init&gt;(JDBC4Connection
.java:46)
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance
0(Native Method)
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance
(Unknown Source)
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(Unknown Source)
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionI
mpl.java:352)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegist
eringDriver.java:282)
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)
[10/17/10 12:48 AM]:    at server.util.SQL.createConnection(SQL.java:17)
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:34)
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:54)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsEx
ception: Communications link failure

The last packet successfully received from the server was 1,287,290,927,046 mill
iseconds ago.  The last packet sent successfully to the server was 0 millisecond
s ago.
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance
(Unknown Source)
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(Unknown Source)
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException
(SQLError.java:1074)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:341)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI
mpl.java:2137)
[10/17/10 12:48 AM]:    ... 20 more
[10/17/10 12:48 AM]: Caused by: java.net.ConnectException: Connection refused: c
onnect
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.socketConnect(Native Method)

[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.doConnect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connectToAddress(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.SocksSocketImpl.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.&lt;init&gt;(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.&lt;init&gt;(Unknown Source)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.StandardSocketFactory.connect(Standard
SocketFactory.java:253)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:290)
[10/17/10 12:48 AM]:    ... 21 more
[10/17/10 12:48 AM]: java.lang.NullPointerException
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:29)
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:54)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: java.lang.NullPointerException
[10/17/10 12:48 AM]:    at server.util.SQL.destroyConnection(SQL.java:42)
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:33)
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:55)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Co
mmunications link failure

The last packet successfully received from the server was 1,287,290,928,031 mill
iseconds ago.  The last packet sent successfully to the server was 0 millisecond
s ago.
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance
(Unknown Source)
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(Unknown Source)
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException
(SQLError.java:1074)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI
mpl.java:2214)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.&lt;init&gt;(ConnectionImpl.j
ava:773)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.JDBC4Connection.&lt;init&gt;(JDBC4Connection
.java:46)
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance
0(Native Method)
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance
(Unknown Source)
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(Unknown Source)
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionI
mpl.java:352)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegist
eringDriver.java:282)
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)
[10/17/10 12:48 AM]:    at server.util.SQL.createConnection(SQL.java:17)
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:34)
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:55)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsEx
ception: Communications link failure

The last packet successfully received from the server was 1,287,290,928,031 mill
iseconds ago.  The last packet sent successfully to the server was 0 millisecond
s ago.
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance
(Unknown Source)
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst
ance(Unknown Source)
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException
(SQLError.java:1074)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:341)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI
mpl.java:2137)
[10/17/10 12:48 AM]:    ... 20 more
[10/17/10 12:48 AM]: Caused by: java.net.ConnectException: Connection refused: c
onnect
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.socketConnect(Native Method)

[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.doConnect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connectToAddress(Unknown Sou
rce)
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.SocksSocketImpl.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.&lt;init&gt;(Unknown Source)
[10/17/10 12:48 AM]:    at java.net.Socket.&lt;init&gt;(Unknown Source)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.StandardSocketFactory.connect(Standard
SocketFactory.java:253)
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.&lt;init&gt;(MysqlIO.java:290)
[10/17/10 12:48 AM]:    ... 21 more
[10/17/10 12:48 AM]: java.lang.NullPointerException
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:29)
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:55)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: java.lang.NullPointerException
[10/17/10 12:48 AM]:    at server.util.SQL.destroyConnection(SQL.java:42)
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:389)
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP
acket(ClickingButtons.java:638)
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack
etHandler.java:138)
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie
nt.java:970)
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand
ler.java:136)
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)
[10/17/10 12:48 AM]: Game saved for player Dclaws Ko
[10/17/10 12:48 AM]: [DEREGISTERED]: Dclaws Ko
[/code]

can anybody help me fix it?</description>
    
    <lastBuildDate>Mon, 18 Oct 2010 00:53:24 +0000</lastBuildDate>
    <category>Runescape</category>
    <atom:link href="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>[pi] server cmd started saying this</title>
        <dc:creator><![CDATA[@Scape-JAVA Scape-JAVA]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/scape-java">@Scape-JAVA</a> wrote:</p>
          <blockquote>
              <p>Alrighty, well the problem is fixed so I see no need for this topic to remain open.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/5">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/5</link>
        <pubDate>Mon, 18 Oct 2010 00:53:24 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-364851-5</guid>
        <source url="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851.rss">[pi] server cmd started saying this</source>
      </item>
      <item>
        <title>[pi] server cmd started saying this</title>
        <dc:creator><![CDATA[@deans1243 deans1243]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/deans1243">@deans1243</a> wrote:</p>
          <blockquote>
              <p>oh i figured it out. i added highscores to forums and i took it off so it couldnt find the link or whatever. i fixed it:) this can be closed now.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/4">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/4</link>
        <pubDate>Mon, 18 Oct 2010 00:46:17 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-364851-4</guid>
        <source url="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851.rss">[pi] server cmd started saying this</source>
      </item>
      <item>
        <title>[pi] server cmd started saying this</title>
        <dc:creator><![CDATA[@EmulousGuru EmulousGuru]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/emulousguru">@EmulousGuru</a> wrote:</p>
          <blockquote>
              <p>Looks like it can’t establish a connection.</p>
<p>Just comment out the MySQL connector from the Server’s initialize method inside the Server class.</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/3">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/3</link>
        <pubDate>Mon, 18 Oct 2010 00:38:28 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-364851-3</guid>
        <source url="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851.rss">[pi] server cmd started saying this</source>
      </item>
      <item>
        <title>[pi] server cmd started saying this</title>
        <dc:creator><![CDATA[@Rixxx Rix_]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/rixxx">@Rixxx</a> wrote:</p>
          <blockquote>
              <p>Something is wrong with your SQL!</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/2">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/2</link>
        <pubDate>Mon, 18 Oct 2010 00:37:09 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-364851-2</guid>
        <source url="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851.rss">[pi] server cmd started saying this</source>
      </item>
      <item>
        <title>[pi] server cmd started saying this</title>
        <dc:creator><![CDATA[@deans1243 deans1243]]></dc:creator>
        <description><![CDATA[
          <p><a href="https://forum.moparisthebest.com/u/deans1243">@deans1243</a> wrote:</p>
          <blockquote>
              <p>so when i log out the client screen freezes and the server cmd says this. the player char file will save and all but it freezes and you have to restart client for you to log back on.</p>
<p>[code]acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsEx<br>
ception: Communications link failure</p>
<p>The last packet successfully received from the server was 1,287,290,926,062 mill<br>
iseconds ago.  The last packet sent successfully to the server was 0 millisecond<br>
s ago.<br>
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance<br>
(Unknown Source)<br>
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException<br>
(SQLError.java:1074)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.(MysqlIO.java:341)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI<br>
mpl.java:2137)<br>
[10/17/10 12:48 AM]:    … 20 more<br>
[10/17/10 12:48 AM]: Caused by: java.net.ConnectException: Connection refused: c<br>
onnect<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.socketConnect(Native Method)</p>
<p>[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.doConnect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connectToAddress(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.SocksSocketImpl.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.(Unknown Source)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.StandardSocketFactory.connect(Standard<br>
SocketFactory.java:253)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.(MysqlIO.java:290)<br>
[10/17/10 12:48 AM]:    … 21 more<br>
[10/17/10 12:48 AM]: java.lang.NullPointerException<br>
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:29)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:53)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: java.lang.NullPointerException<br>
[10/17/10 12:48 AM]:    at server.util.SQL.destroyConnection(SQL.java:42)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:33)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:54)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Co<br>
mmunications link failure</p>
<p>The last packet successfully received from the server was 1,287,290,927,046 mill<br>
iseconds ago.  The last packet sent successfully to the server was 0 millisecond<br>
s ago.<br>
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance<br>
(Unknown Source)<br>
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException<br>
(SQLError.java:1074)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI<br>
mpl.java:2214)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.(ConnectionImpl.j<br>
ava:773)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.JDBC4Connection.(JDBC4Connection<br>
.java:46)<br>
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
0(Native Method)<br>
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
(Unknown Source)<br>
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionI<br>
mpl.java:352)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegist<br>
eringDriver.java:282)<br>
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.createConnection(SQL.java:17)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:34)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:54)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsEx<br>
ception: Communications link failure</p>
<p>The last packet successfully received from the server was 1,287,290,927,046 mill<br>
iseconds ago.  The last packet sent successfully to the server was 0 millisecond<br>
s ago.<br>
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance<br>
(Unknown Source)<br>
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException<br>
(SQLError.java:1074)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.(MysqlIO.java:341)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI<br>
mpl.java:2137)<br>
[10/17/10 12:48 AM]:    … 20 more<br>
[10/17/10 12:48 AM]: Caused by: java.net.ConnectException: Connection refused: c<br>
onnect<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.socketConnect(Native Method)</p>
<p>[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.doConnect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connectToAddress(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.SocksSocketImpl.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.(Unknown Source)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.StandardSocketFactory.connect(Standard<br>
SocketFactory.java:253)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.(MysqlIO.java:290)<br>
[10/17/10 12:48 AM]:    … 21 more<br>
[10/17/10 12:48 AM]: java.lang.NullPointerException<br>
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:29)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:54)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: java.lang.NullPointerException<br>
[10/17/10 12:48 AM]:    at server.util.SQL.destroyConnection(SQL.java:42)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:33)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:55)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Co<br>
mmunications link failure</p>
<p>The last packet successfully received from the server was 1,287,290,928,031 mill<br>
iseconds ago.  The last packet sent successfully to the server was 0 millisecond<br>
s ago.<br>
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance<br>
(Unknown Source)<br>
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException<br>
(SQLError.java:1074)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI<br>
mpl.java:2214)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.(ConnectionImpl.j<br>
ava:773)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.JDBC4Connection.(JDBC4Connection<br>
.java:46)<br>
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
0(Native Method)<br>
[10/17/10 12:48 AM]:    at sun.reflect.NativeConstructorAccessorImpl.newInstance<br>
(Unknown Source)<br>
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionI<br>
mpl.java:352)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegist<br>
eringDriver.java:282)<br>
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.sql.DriverManager.getConnection(Unknown Source)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.createConnection(SQL.java:17)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:34)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:55)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsEx<br>
ception: Communications link failure</p>
<p>The last packet successfully received from the server was 1,287,290,928,031 mill<br>
iseconds ago.  The last packet sent successfully to the server was 0 millisecond<br>
s ago.<br>
[10/17/10 12:48 AM]:    at sun.reflect.GeneratedConstructorAccessor2.newInstance<br>
(Unknown Source)<br>
[10/17/10 12:48 AM]:    at sun.reflect.DelegatingConstructorAccessorImpl.newInst<br>
ance(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.lang.reflect.Constructor.newInstance(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.SQLError.createCommunicationsException<br>
(SQLError.java:1074)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.(MysqlIO.java:341)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionI<br>
mpl.java:2137)<br>
[10/17/10 12:48 AM]:    … 20 more<br>
[10/17/10 12:48 AM]: Caused by: java.net.ConnectException: Connection refused: c<br>
onnect<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.socketConnect(Native Method)</p>
<p>[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.doConnect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connectToAddress(Unknown Sou<br>
rce)<br>
[10/17/10 12:48 AM]:    at java.net.PlainSocketImpl.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.SocksSocketImpl.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.connect(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.(Unknown Source)<br>
[10/17/10 12:48 AM]:    at java.net.Socket.(Unknown Source)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.StandardSocketFactory.connect(Standard<br>
SocketFactory.java:253)<br>
[10/17/10 12:48 AM]:    at com.mysql.jdbc.MysqlIO.(MysqlIO.java:290)<br>
[10/17/10 12:48 AM]:    … 21 more<br>
[10/17/10 12:48 AM]: java.lang.NullPointerException<br>
[10/17/10 12:48 AM]:    at server.util.SQL.query(SQL.java:29)<br>
[10/17/10 12:48 AM]:    at server.util.SQL.saveHighScore(SQL.java:55)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:388)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: java.lang.NullPointerException<br>
[10/17/10 12:48 AM]:    at server.util.SQL.destroyConnection(SQL.java:42)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.logout(Client.java:389)<br>
[10/17/10 12:48 AM]:    at server.model.players.packets.ClickingButtons.processP<br>
acket(ClickingButtons.java:638)<br>
[10/17/10 12:48 AM]:    at server.model.players.PacketHandler.processPacket(Pack<br>
etHandler.java:138)<br>
[10/17/10 12:48 AM]:    at server.model.players.Client.processQueuedPackets(Clie<br>
nt.java:970)<br>
[10/17/10 12:48 AM]:    at server.model.players.PlayerHandler.process(PlayerHand<br>
ler.java:136)<br>
[10/17/10 12:48 AM]:    at server.Server.main(Server.java:151)<br>
[10/17/10 12:48 AM]: Game saved for player Dclaws Ko<br>
[10/17/10 12:48 AM]: [DEREGISTERED]: Dclaws Ko<br>
[/code]</p>
<p>can anybody help me fix it?</p>
          </blockquote>
          <p><a href="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/1">Read full topic</a></p>
        ]]></description>
        <link>https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851/1</link>
        <pubDate>Mon, 18 Oct 2010 00:35:41 +0000</pubDate>
        <guid isPermaLink="false">forum.moparisthebest.com-post-364851-1</guid>
        <source url="https://forum.moparisthebest.com/t/pi-server-cmd-started-saying-this/364851.rss">[pi] server cmd started saying this</source>
      </item>
  </channel>
</rss>
